types/base/api.d.ts

```diff
@@ -0,0 +1,42 @@
+type Part = {
+    name: string;
+    material: string;
+    color: string;
+}
+
+type Spec = {
+    brand: string;
+    model: string;
+    parts: Array<Part>;
+    initials: string;
+    engraving: string;
+    initialsExtra: object;
+    variant?: string;
+    version?: string;
+    description?: string;
+};
+
+type ProductionOrderOptions = {
+    brand?: string;
+    model?: string;
+    variant?: string;
+    parts?: Array<Part>;
+    initials?: string;
+    engraving?: string;
+    initialsExtra?: object;
+    gender?: string;
+    size?: number;
+    pending?: boolean;
+    notify?: boolean;
+    productId?: string;
+    currency?: string;
+    country?: string;
+    meta?: Array<string>;
```

Comment:
Isn't this an object?

Reply:
Does not seem like it:
https://github.com/ripe-tech/ripe-sdk/blob/09f547ab354bfad8fb302bacc69cd2223770f988/src/js/api/order.js#L653